src/api/data/cache/charthelper/chart_package_helper.go

```diff
@@ -29,11 +29,13 @@ var DownloadAndExtractChartTarball = func(chart *models.ChartPackage) error {
 
 	if !tarballExists(chart) {
 		if err := downloadTarball(chart); err != nil {
+			cleanChartDataDir(chart)
```

Comment:
Oh nice, I was trying to figure out how I could do this, will change it to this - thanks!